src/Common/src/Interop/Unix/libssl/SecuritySafeHandles.cs

```diff
@@ -266,6 +276,8 @@ public override bool IsInvalid
         protected override bool ReleaseHandle()
         {
             Interop.OpenSsl.FreeSslContext(_sslContext);
+            _sslContext.DangerousRelease();
+            _sslContext.Dispose();
```

Comment:
I see.  Is FreeSslContext still necessary then, or can it be removed and call sites changed to just call Disconnect, since that's all FreeSslContext does now?